src/main/java/nus/climods/logic/parser/CliModsParser.java

```diff
@@ -40,13 +41,13 @@ public static Command parseCommand(String userInput) throws ParseException {
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
-        case ListCommand.COMMAND_WORD:
+        case (ListCommand.COMMAND_WORD):
             String trimmedArgs = arguments.trim();
```

Comment:
@SeekSaveServe I just realised this was here, can we not do this here? But delegate it to a `ListCommandParser` instead? Its bad DX when we have command parsers for some commands and not for some.

Reply:
oh i removed ListCommandParser ystd cos it was just dumping the args into the parameter. I can add it back

Reply:
@shenyih0ng, i've added back the ListCommandParser!